### PR TITLE
BC-2945: set dev ES_DOMAIN to new kubernetes edusharing dev instance

### PR DESCRIPTION
**Pull request by ssmid**
# Description
Switch to new K8 Edu-Sharing

## Links to Tickets or other pull requests
Ticket: https://ticketsystem.dbildungscloud.de/browse/BC-2945

https://github.com/hpi-schul-cloud/schulcloud-server/pull/3810

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by virgilchiriac — CHANGES_REQUESTED** [review]

**Review by Loki-Afro — CHANGES_REQUESTED** [review]
see comments

**Comment by virgilchiriac** [comment]
is this the production K8 ES instance? Do we have staging / dev instances?

File: ansible/group_vars/all/config.yml [hunk 1]
```diff
@@ -591,8 +591,13 @@ configuration_all:
...
     client: true
     nuxtclient: false
+  ES_API_V7:
+    value: "false"
+    server: true
+    client: false
+    nuxtclient: false
   ES_DOMAIN:
-    value: "https://mv-repo.dbildungscloud.de"
+    value: "https://repository.sc-dev-edusharing.dbildungscloud.dev"
```
[the remark this answers is not in the document]
Comment (dimapin) [reply ansible/group_vars/all/config.yml]: Is set to http://repository.edusharing.dbildungscloud.de/

**Comment by dimapin** [comment]
> is this the production K8 ES instance? Do we have staging / dev instances?

Yes, https://repository.sc-staging-edusharing.dbildungscloud.org/edu-sharing/ and https://repository.sc-dev-edusharing.dbildungscloud.dev/edu-sharing/

**Review by dimapin — APPROVED** [review]

**Review by Loki-Afro — APPROVED** [review]

**Review by Loki-Afro — APPROVED** [review]

**Pull request merged**